kart/cli_util.py

```diff
@@ -12,6 +13,22 @@
 from click.core import Argument
 from click.shell_completion import CompletionItem
 
+from kart.help import get_renderer
+
+
+class KartCommand(click.Command):
+    def format_help(self, ctx, formatter):
+        file = Path(os.path.dirname(sys.prefix)) / os.path.join(
+            "docs", "pages", "commands", f"{ctx.command.name}.rst"
+        )
+        with file.open("a") as f:
```

Comment:
Not sure what this is for? Looks like you are appending the empty string to a file

Reply:
Also if you are using pathlib.Path, you can mostly use `write_text` etc instead of open() - a bit simpler to use

Reply:
AFAIK, `write_text` overwrites the file contents. Since we're reading from the docs here to produce the man pages, appending would be better.

Reply:
Updated.